src/core/api/fims/admin/policy.xml

```diff
@@ -0,0 +1,15 @@
+<policies>
```

Comment:
I'd replace the `set-backend-service` with the API group `service-url` property

Reply:
admin should be private right?